src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -469,7 +469,7 @@ public boolean isRegenerateRequired() {
 
 	// Below are methods that may still be used by some plugins
 
-	public void loadStoredDataFrom(Workspace other) {
+	@Deprecated public void loadStoredDataFrom(Workspace other) {

```

Comment:
Deprecated may not be the best word as the method is still relevant. How about making it package private and then making "adapter" class in your plugin with same package "net.mcreator.workspace" that exposes those methods to your plugin?

Reply:
Does that allow to share package-private access?

Reply:
:heavy_check_mark: Fixed

Reply:
Just realized this is the only method that assigns something to `Workspace#foldersRoot`. Should this field be refactored to become `private final`? (Even if another PR)

Reply:
If it is ok final for your plugin, you can do it here

Reply:
As you still need to assign it here, it should remain as it is